Orlando Walking Tours/Controllers/LocationListVC.swift

```diff
@@ -80,23 +82,19 @@ class LocationListVC: UIViewController
         tableView.dataSource = self
         tableView.delegate = self
         
-        closestToMe.didTouch =
-        {
-            self.sortClosestToMe()
-            self.tableView.reloadData()
-        }
+//        closestToMe.didTouch =
```

Comment:
Yeah, I left this stuff in because I need to re-implement it.  When I changed the layout, I completely remove the TouchableLabel that handled this functionality.  I will add a TODO: to remind me to work on that.